site/styles/_footer.scss

```diff
@@ -2,6 +2,7 @@
 footer {
 	padding: 10px 10px;
 	flex-direction: column;
+	flex-shrink: 0;
```

Comment:
I'm not familiar with this one, explanation?

Reply:
flex shrink is the opposite of flex grow. When the content is too large for a container, it tells sibling flex items the rate at which they should shrink. Eg. If item1 has a shrink value of 1 and item2 has a shrink value of 2, item2 will shrink twice as much as item1. By default, the value is 1. Changing it to 0 means that flex items will not shrink smaller than their default size. I think what this does is prevent unwanted vertical scaling (remember that .page and footer are now flex items in a column).

Reply:
Thanks for the ELI5! 👍